### DISCUSSION TITLE
Is there a JSON schema validator for the HDF format?

### DISCUSSION THREAD

**Discussion by trevor-vaughan** [mitre/saf#1306] [Q&A] [answered]
I'd like to be able to validate that generated HDF is valid on the fly.

Apologies if I've missed it in the documentation.

**Comment by Amndeep7** [comment, marked as answer]
Hello @trevor-vaughan, apologies for the delay in response.

Unfortunately, we do not currently have a formal schema validator utility that we provide ourselves.  We do provide the schema itself here: https://github.com/mitre/heimdall2/blob/master/libs/inspecjs/schemas/exec-json.json.
